README.md

```diff
@@ -7,6 +7,7 @@ By default, the multicluster controlplane would try to find ocmconfig.yaml in th
 Here is a simple file of config-file.yaml:
 ```yaml
 dataDirectory: "/.ocm"
+serviceType: clusterip
```

Comment:
I think we do not need an additional to indicate this, we can detect this in the controlplane